package.json

```diff
@@ -16,7 +16,13 @@
   "devDependencies": {},
   "scripts": {
     "start": "tsc server/ts/* --outDir server/js --lib es6,dom && tsc client/ts/* --outDir client/js --lib es6,dom && node server/js/main.js",
-    "test": "echo \"Error: no test specified\" && exit 1"
+	 "test": "echo \"Error: no test specified\" && exit 1",
+	 "dev": "npm run tsc && npm run cp-html && node build/server/server.js",
+	 "cp-html": "cp src/client/**.html build/client",
+	 "tsc": "npm run tsc-client && npm run tsc-server",
```

Comment:
Just a suggestion, 'npm run compile' (and similarly compile-server, compile-client) might be more readable names.